Feature/explorador_empresas

ra247362 opened the pull request:
Implementa o explorador de empresas como parte do _app Search_.

Closes #21

ra247362 committed:
feature(search/explorer): reworked explorer into search app

Moved explorer functionality from Company to Search, allowing for possible reuse of filters. Reworked internal app structure to fit changes.

ra247362 committed:
chore(search/explorer): removed unused leftover code

Removed unused code from Company that was left over from the Explorer migration.

ra247362 committed:
chore(search/explorer): add update test

Adds an update acceptance test, as per user story. Checks if explorer is properly updated after a change in the database.

Lozavival commented on search/views.py:
Os métodos get_queryset e get, na classe ExplorerView, não parecem fazer nada novo ou alterar o comportamento padrão da superclasse

ra247362 replied:
Corrigido.

EDIT: Agora corrigido de fato, sem os comentários. Peço perdão pelo ocorrido.

ra247362 committed:
chore(search/explorer): minor fixes

Removed unnecessary code from ExplorerView and changed convertion from QuerySet to list in ExplorerTestCase.

ra247362 committed:
chore(search/explorer): minor fix

Deleted commented lines in views.py.

Lozavival reviewed (approved):
Tudo certo, aprovado!
